shared/decoders.ts

```diff
@@ -323,4 +323,67 @@ export function decodeMicroGrantsHatsInitializedData(
     hatsContract,
     hatsId
   }
+}
+
+
+// ===========================
+// = SQFSuperFluids Decoder  =
+// ===========================
+
+export function decodeSQFSuperFluidInitializedData(data: any) {
+  const [
+    useRegistryAnchor,
+    metadataRequired,
+    passportDecoder,
+    superfluidHost,
+    allocationSuperToken,
+    registrationStartTime,
+    registrationEndTime,
+    allocationStartTime,
+    allocationEndTime,
+    minPassportScore,
+    initialSuperAppBalance
+  ] = decodeAbi(data, [
+      "bool",
+      "bool",
+      "address",
+      "address",
+      "address",
+      "uint64",
+      "uint64",
+      "uint64",
+      "uint64",
+      "uint256",
+      "uint256",
+  ])
+
+  return {
+    useRegistryAnchor,
+    metadataRequired,
+    passportDecoder,
```

Comment:
I have a todo in the pipeline to auto-lowercase decoded `address` types, since all addresses on Spec are lowercase. But until that update is out, let's just do something like this to be sure:
```typescript
passportDecoder: passportDecoder?.toLowerCase(),
superfluidHost: superfluidHost?.toLowerCase(),
allocationSuperToken: allocationSuperToken?.toLowerCase(),
```